cfpb/consumerfinance.gov
# Update `update` icon to animated `updating` icon

We need a canonical updating icon, which I've added in https://github.com/cfpb/design-system/pull/1610

This PR reverts some changes in https://github.com/cfpb/consumerfinance.gov/pull/7558 to use the canonical `updating` icon in place of `update`.

## Changes

- Update `update` icon to animated `updating` icon.
- Update ROUT tool to use `svg_icon(…)` helper in place of hardcoded icon.

## How to test this PR

1. Visit http://localhost:8000/rural-or-underserved-tool/ and enter an address and click "Check addresses" and see that there is an animated icon.
2. Visit http://localhost:8000/consumer-tools/retirement/before-you-claim/ and enter details and click "Get your estimates" and see that there's an animated icon in the button (there's an unrelated issue of the button becoming bigger temporarily).
3. Visit http://localhost:8000/privacy/records-access/ and either trust it's good or submit a test request and see that submit button has an animated icon.
